tests/lexer/tcustom_numeric_literals.nim

```diff
@@ -165,21 +162,16 @@ template main =
     doAssert fn2() == "[[-12]]"
     doAssert fn3() == "[[-12]]"
 
-    when false: # xxx this fails; bug 9 from https://github.com/nim-lang/Nim/pull/17020#issuecomment-803193947
```

Comment:
Meanwhile this was actually the bug fixed by this PR, custom numeric literals generate dot fields which did not keep their resolved symbols in templates. It really wouldn't have taken much investigation to figure out this is related to dot fields and has nothing to do with custom numeric literals. The linked comment thinks it has something to do with `'abc` not working where `abc` is a template param??? Very weird understanding of the language